src/main/java/bio/terra/service/snapshot/SnapshotService.java

```diff
@@ -167,8 +173,24 @@ public SnapshotSummaryModel retrieveSnapshotSummary(UUID id) {
      * @return a SnapshotModel = API output-friendly representation of the Snapshot
      */
     public SnapshotModel retrieveAvailableSnapshotModel(UUID id) {
+        return retrieveAvailableSnapshotModel(id, getDefaultIncludes());
+    }
+
+    /**
+     * Convenience wrapper around fetching an existing Snapshot object and converting it to a Model object.
+     * Unlike the Snapshot object, the Model object includes a reference to the associated cloud project.
+     *
```

Comment:
javadoc ignores whitespace so you need a paragraph tag to get paragraph breaks
```suggestion
     * <p>
```

Reply:
Ah shoot right